modules/ROOT/pages/security.adoc

```diff
@@ -66,6 +72,28 @@ Use if you authenticate to your API using custom header parameters and values.
 
 Any other authentication methods to access your added API, such as custom OAuth implementations or short-lived token methods, are not supported.
 
+== Configuring Mutual Authentication in DataGraph
+
+Mutual authentication (mTLs) ensures that Datagraph and the underlying APIs in your unified schema can create secure, verified connections.
+
+If an API requires mTLS, you must configure mTLS when adding it to the unified schema, or when promoting it to another environment. To do so, you must provide the following:
```

Comment:
```suggestion
If an API requires mTLS, configure mTLS when adding the API to the unified schema or when promoting the API to another environment. To do so, provide the following:
```
This is just one example of removing the modal "must," per the style ref. If you want emphasis, it's okay, but it's better to remove it because other languages do not use modal verbs. Consider searching for "must" and reevaluating each case.